kytos/core/auth.py

```diff
@@ -56,12 +162,19 @@ def __init__(self, controller):
             controller(kytos.core.controller): A Controller instance.
 
         """
+        self.user_controller = self.get_user_controller()
+        self.user_controller.bootstrap_indexes()
```

Comment:
We'll need to strategically change how this class is being instantiated on Kytos `Controller`, it's OK to have the index bootstrap and getting `UserController` with a `Mongo()` instance here, but we'll need to adapt how it's being initialized, otherwise there will be cases where the DB isn't ready yet, the core provide guarantees but only at certain initialization points. 

Let's initialize `self.auth = Auth()` and register its endpoints only when [`start()`](https://github.com/kytos-ng/kytos/blob/master/kytos/core/controller.py#L245) is called after the DB has been waited, you can a new method on Controller, `start_auth()` can called it there and init `self.auth = Auth()` on this new method and maintain this constructor here as it is, but also need to move [`self.auth.register_core_auth_services()`](https://github.com/kytos-ng/kytos/blob/6a5e02f766ff2961a0cc7554059eb1b5a07e4679/kytos/core/controller.py#L409) as well to lazily initialize the auth endpoints.

Reply:
Oh, ok.